Projects/Server/main.cpp

```diff
@@ -8,19 +8,35 @@
 
 int main()
 {
+    size_t p0win{ 0 };
+    size_t p1win{ 0 };
+    size_t draw{ 0 };
+    size_t gameSample{ 500 };
```

Comment:
Please keep these in a struct, these are related data.

Reply:
Made a map out of them in the newly created class.